plugin.rb

```diff
@@ -18,3 +18,11 @@
 require_relative "lib/linked_in_authenticator"
 
 auth_provider authenticator: ::LinkedInAuthenticator.new, icon: "fab-linkedin"
+
+after_initialize do
+  if SiteSetting.linkedin_enabled
+    AdminDashboardData.add_problem_check do
```

Comment:
It looks like we are using a deprecated method? 

https://github.com/discourse/discourse/blob/1f73e7d039c94be5aaf71a225c4f7bdd9ac82668/app/models/admin_dashboard_data.rb#L46

Will this create more work for us in the future to remove this deprecation?

Reply:
Ah. Yes. Ironically I am the one who deprecated it. 😅